src/includes/set-fingerprint/rpc/apple.mdx

```diff
@@ -0,0 +1,39 @@
+```swift
+import Sentry
+
+SentrySDK.start { options in
+    options.beforeSend = { event in
+        if let error = event.error as NSError? {
+            if let failingUrl = error.userInfo[NSURLErrorFailingURLErrorKey] as? String,
+               error.domain == NSURLErrorDomain {
```

Comment:
```suggestion
        if let error = event.error as? NSError, error.domain == NSURLErrorDomain {
                    if let failingUrl = error.userInfo[NSURLErrorFailingURLErrorKey] as? String {
```

I would group the error domain check with the error casting.
Move "?" from the type (NSError?) to the cast (as?), is much safer.